Envfile

```diff
@@ -117,8 +117,8 @@ variable :FASTLY_API_KEY, :String, default: "Optional"
 variable :FASTLY_SERVICE_ID, :String, default: "Optional"
 
 # Honeycomb for monitoring and observability
-# https://www.honeycomb.io/
-variable :HONEYCOMB_API_KEY, :String, default: "Optional"
```

Comment:
I kinda like having this default "labeled" Optional since it is.

Reply:
The thing that is weird to me about this is that the variable at runtime will have the value "Optional", which would make a presence check not functional:

```shell
➜  devto git:(master) rails c
DEPRECATION WARNING: Default values will be removed in the next minor-release of ENVied (i.e. > v0.9). For more info see https://gitlab.com/envied/envied/tree/0-9-releases#defaults. (called from block in load at /Users/rhymes/Development/devto/Envfile:4)
Libhoney::Client: no writekey configured, disabling sending events
Libhoney::Client: no writekey configured, disabling sending events
Loading development environment (Rails 5.2.3)
[1] pry(main)> ApplicationConfig["HONEYCOMB_API_KEY"]
=> "Optional"
```

Reply:
Ah that's right, Mac was telling me about that